### DISCUSSION TITLE
Firmware Update für Wechselrichter

### DISCUSSION THREAD

**Discussion by markusruebe** [tbnobody/OpenDTU#1841] [Ideas]
Hallo zusammen, gibt es die Möglichkeit, über die OpenDTU die Firmware eines Hoymiles WR zu aktualisieren?
Geht das über die DTU oder braucht man einen hoymiles account incl. dem Cloud Gedöns?

Viele Grüße

**Comment by MetaChuh** [comment]
@markusruebe 

ja, derzeit ist kein inverter firmware update, bzw länderspezifischen grid profiles, via opendtu, sowie keinem anderen uns derzeit bekannten, privaten dtu projekten möglich.

das cloud gedöns ist aus mehreren standpunkten nach wie vor derzeit notwendig.
(sidenote: notwendig, wenn wir nicht kamikaze spielen wollen)

diese sind zum beispiel:
hersteller haftung gilt nur, wenn man alle vom hersteller empfohlene schritte und wege, ohne abweichung, durchführt.
die schadenshaftung seitens des herstellers exkludiert jegliche haftbarkeit, bei nicht offizieller veränderung ihrer geräte.

greetings,
metachuh
